### PR TITLE
ASM Generator

### PR DESCRIPTION
**Pull request by uslsteen**
1. genAsm RISC-V opcodes added

### CODE REVIEW

File: isa/info.py [hunk 1]
```diff
@@ -30,6 +30,10 @@ class InstrFormatTy(IntEnum):
     InstrFormatTy.J: ({"rd", "jimm20"},),
 }
 
+INSTR_CATEGORY = {
+    "LOAD_CATEGORY": ["LB", "LBU", "LH", "LHU", "LW"],
```
Comment (derzhavin3016) [inline isa/info.py]: Please, use values of type InstrNameTy

File: generator.py [hunk 2]
```diff
@@ -26,3 +34,44 @@ def regGen():
...
+            #
+            case InstrFormatTy.U: 
+                asm_str = '{} x{}, {}'.format(
+                    mnemonic, instr_data.rd, hex(instr_data.imm))
+            #
+            case InstrFormatTy.J:
+                asm_str = '{} x{}, {}'.format(
+                    mnemonic, instr_data.rd, hex(instr_data.imm))
+            #
+            case _ : print("Error : there is no support assembler of systsem instruction")
```
Comment (derzhavin3016) [inline generator.py]: Please, raise an exception here

File: generator.py [hunk 3]
```diff
@@ -4,13 +4,21 @@
...
+from isa.info import InstrFormatTy
 
 class Generator:
     def __init__(self):
         pass
 
+    def generate(self):
+        l = []
+        l.extend([instr.generate() for i in range(100)])
+        return l
```
Comment (Bryanskiy) [inline generator.py]: unused

**Comment by Bryanskiy** [comment]
Please, rebase from master, your commits are REALLY outdated

**Comment by Bryanskiy** [comment]
Please, write also an RFC about this proposal

**Comment by derzhavin3016** [comment]
@Bryanskiy , thank you for your responsible monitoring of this repository to follow our corporate rules!!

**Comment by derzhavin3016** [comment]
I suppose that without any kind of documented proposal, we cannot merge this feature (too much undocumented code w/out any rationale)

File: generator.py (hunk 3 above)
Comment (Bryanskiy) [reply generator.py]: @Bryanskiy outstanding observation, thank you for your professional review of this PR!!!!
I hope that @uslsteen will fix it ASAP

**Comment by Bryanskiy** [comment]
Please, stay synced w/ out recent updates on master, or you'll never merge this PR....

**Pull request merged**